docs/src/documentation/03-components/05-input/inputfield/03-accessibility.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/inputfield/accessibility/
+---
+
+### Accessibility
+
+- For special cases you can use your own, detached `label`. Simply like this:
+
+```jsx
+<label for="NICEID">Content</label>
+<InputField
+  id="NICEID"
+/>
+```
+
+- The `ariaLabel` prop allows you to specify an `aria-label` attribute for the InputField component. This attribute provides additional information to screen readers, enhancing the accessibility of the component. By using `ariaLabel`, you can ensure that users who rely on assistive technologies receive the necessary context and information about the component's purpose and functionality.
+- If the `label` prop is not provided, the `ariaLabel` prop must be specified to ensure component accessibility.
```

Comment:
This should be ensured by typescript, imo. It will be a breaking change.

Reply:
Hmm, I don't understand. I've just moved the text from README to a new file. :D I didn't add any ariaLabel or label, so there shouldn't be any BC in this PR.

Reply:
I know what you did 😄 I am saying that:
> If the `label` prop is not provided, the `ariaLabel` prop must be specified

should probably be controlled by Typescript (similarly to what we have in other components). I mean: have the `ariaLabel` prop be required if `label` is undefined. If we do that, it is a breaking change

Reply:
As discussed offline, we won't do it now